bigcache.go

```diff
@@ -134,6 +134,13 @@ func (c *BigCache) Set(key string, entry []byte) error {
 	return shard.set(key, hashedKey, entry)
 }
 
+// Append appends entry under existing key
```

Comment:
Can you elaborate this? What if key does not exist and how it differs from `Set`

Reply:
pls check